docs/development/language.md

```diff
@@ -0,0 +1,68 @@
+ILIAS Language Handling
+=======================
+ILIAS offers multi-language support for the interface of ILIAS.
+
+# Guidelines
+1.  All language entries are text strings and stored in language files in the 
+    subdirectory `/lang`. Each language entry has the format:
+
+        `language_module_ID#:#variable_ID#:#text_content###comment`
+
+    The elements `language_module_ID`, `variable_ID` and `variable_ID`  are required, 
+    while `comment` is optional and can be used for additional information about an 
+    entry, e.g. „`07 02 2020 new variable`“.
+    
+2. The `variable_ID` of a language entry MUST be unique within the whole language file. This avoids conflicts in the 
+presentation of language entries because the `language_module_ID` is not taken into consideration when ILIAS inserts 
+language entries into the HTML output.
```

Comment:
IMO we should remove `HTML` here, because the translations are also use in andy kind of export (PDF, CSV, Excel, ...).

Reply:
I would tend to agree:
```suggestion
language entries into the output.
```

Reply:
I fully agree.